src/msw/webview_ie.cpp

```diff
@@ -853,25 +857,229 @@ wxString wxWebViewIE::GetPageText() const
     }
 }
 
-void wxWebViewIE::RunScript(const wxString& javascript)
+wxWebViewIEEmulationLevel wxWebViewIE::GetEmulationLevel()
```

Comment:
I am not 100% sure but I believe that names for methods available only on Windows start with "MSW"? Perhaps @sjlamerton knows more about this...